cms/djangoapps/contentstore/tests/test_proctoring.py

```diff
@@ -0,0 +1,254 @@
+"""
+Tests for the edx_proctoring integration into Studio
+"""
+
+from mock import patch
+
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+
+from contentstore.signals import listen_for_course_publish
+
+from edx_proctoring.api import get_all_exams_for_course
+
+
+@patch.dict('django.conf.settings.FEATURES', {'ENABLE_PROCTORED_EXAMS': True})
+class TestProctoredExams(ModuleStoreTestCase):
+    """
+    Tests for the publishing of proctored exams
+    """
+
+    def setUp(self):
+        """
+        Initial data setup
+        """
+        super(TestProctoredExams, self).setUp()
+
+        self.course = CourseFactory.create(
+            org='edX',
+            course='900',
+            run='test_run',
+            enable_proctored_exams=True
+        )
+
+    def test_publishing_proctored_exam(self):
```

Comment:
hmmm. I've never used DDT before. I'll schedule this for last and try to find relevant examples. This is a "must have" or "nice to have".